### DISCUSSION TITLE
<about flag>

### DISCUSSION THREAD

**Discussion by koki0608pon** [community/community#45796] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

To GitHub Support Team,

When I logged in to GitHub, I got the following message.
“Your account has been flagged.”

I don’t know why my account has been flagged, and I think this is a mistake.

Could you remove this flag, please?

Regards,
<Koki>

**Comment by cehfisher** [comment]
Hi @koki0608pon - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to a different category? Thanks!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @koki0608pon ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
